velox/exec/tests/PrestoQueryRunnerTest.cpp

```diff
@@ -32,6 +32,10 @@ namespace facebook::velox::exec::test {
 class PrestoQueryRunnerTest : public ::testing::Test,
                               public velox::test::VectorTestBase {
  protected:
+  static void SetUpTestCase() {
+    memory::MemoryManager::testingSetInstance({});
```

Comment:
Why is this needed? CC: @xiaoxmeng

Reply:
The test fails claiming memory manager not initialized without this.